src/doc/gui_manual/Operators/OperatorTypes/PICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,196 @@
+Poincaré section. To limit run-a-way integration, the user can control the
+termination criteria.
+
+For the integral curves that return to the Poincaré section a signed return
+distance is calculated. Curves with a return distance below the cycle tolerance
+are considered to be limit cycles. If a curve does not satisfy the tolerance,
+then its return distance is compared to its neighboring integral curves. If a
+zero crossing is found, then a binary search is conducted. The binary search is
+also limited by a maximum number of iterations.
+
+
+Source
+^^^^^^
+
+Source Type
+"""""""""""
+
+The user can seed the integral curves using samples on a line. This line
+defines a Poincaré section used to detect limit cycles. Source options are:
+
+Line
+    Source is specified with the line end points. The line location can be set
+    interactively using the interactive line tool. Options are:
+    
+    * Start - X Y Z location of the line starting point.
+    * End - X Y Z location of the line ending point.
+    
+Sampling type
+"""""""""""""
+
+The Limit Cycle operator only supports uniform samples from the line.
+
+Samples along line
+    Number of uniform samples to take along the line defined above.
+
+
+Field
+"""""
+
+See Field attributes that are common to all PICS operators.
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for numerical integrators. In addition to the Integration
+attributes common to all PICS operators, the Limit Cycle operator supports the
+following attributes.
+
+Integration Direction
+"""""""""""""""""""""
+
+Sets the integration direction through time. Options are:
+    
+Forward
+    Integrate forward in time.
+
+Backward
+    Integrate backward in time.
+
+Both
+    Integrate both forward and backward in time, producing two integral curves.
+
+Forward Directionless
+    Integrate forward in time assuming a directionless vector field.
+
+Backward Directionless
+    Integrate backward in time assuming a directionless vector field.
+
+Both Directionless
+    Integrate both forward and backward in time assuming a directionless vector
+    field, producing two integral curves.
+
+Eigen vectors are an example of a directionless vector field. To integrate
+using a directionless field requires that any orientation discontinuity be
+corrected prior to linear interpolation. That is, all vectors must be rotated 
+to match the orientation of the trajectory. The PICS code will do this
+processing for standard fields (e.g non-higher order elements).
+
+Integrator
+""""""""""
+
+See Integrator attributes that are common to all PICS operators.
+    
+Step Length
+"""""""""""
+
+See Step Length attributes that are common to all PICS operators.
+    
+Tolerances
+""""""""""
+
+See Tolerance attributes that are common to all PICS operators.
+
+Termination
+"""""""""""
+
+Integral curve termination can be controlled in several different ways. The
+termination is based on the most conservative criteria, so only one criteria
```

Comment:
I am reading this to suggest there are *multiple* criteria available it the algorithm uses an *or*'ing of this criteria for termination. Is that right?

Reply:
This is how I understand it as well. I will test to double check.